static/css/layout/index.less

```diff
@@ -13,7 +13,7 @@ div.contentBody,
 div#contentBody {
   padding: 0 @contentBody-padding @contentBody-padding;
   img {
-    max-width: 100%;
+    // max-width: 100%;
```

Comment:
If you want this to be removed please delete it completely instead of commenting it out.